lib/Ama/Controller/Emailfeedback.pm

```diff
@@ -0,0 +1,26 @@
+package Ama::Controller::Emailfeedback;
+use Mojo::Base 'Mojolicious::Controller';
+
+sub submit{
+    my $self = shift;
+    my $sendgrid = $self->sendgrid;
+    my $feedback_comment = $self->param('feedback_comment');
+    #$c -> feedback -> submit($feedback);
+    warn $self -> req -> body;
+    my $send = $sendgrid -> mail(
+        text => $feedback_comment,
+        to      => q(ajin@keystone-it.com),
+        cc      => q(jsiems@keystone-it.com),
+        bcc     => q(bmeyer@keystone-it.com),
```

Comment:
we may want to make this configurable. You can take a look at how I added the vote_floor to ama.conf in my last pull request.

I can help with that on Monday or we can merge this branch and I'll make a new issue that we can push off for later